### PR TITLE
Update faq.asciidoc

### PR DESCRIPTION
**Pull request by jalogisch**
Added the note that the reinstallation of dashboards needs to be done against Kibana and not against elasticsearch API. The DEV Console style indicate that this is against elasticsearch.

### CODE REVIEW

**Comment by jalogisch** [comment]
@EricDavisX I guess I did the PR against the wrong branch - should I create this new against current or will this be used to be pushed to current?

**Comment by EricDavisX** [comment]
Thanks so much for the docs contribution @jalogisch.  I think the note you added needs to have Kibana set as {kib} and Elasticsearch to {es} but otherwise it could be updated to point to 'main' branch and set up with current backport expectations (we usually merge to head of the line then backport).  

I'll request help from @ollyhowell to confirm process, and branch usage and restart this with you, if that's ok Olly?  
Cheers all.

**Review by dedemorton — APPROVED** [review]
LGTM (pending acceptance of suggestion), but I wonder...can't they go to the UI and reinstall the assets from there?

~Nothing here that we'll need to make this change in the `main` branch, too.~ sorry, should have read comments first. :-)

**Comment by dedemorton** [comment]
I get this change into the main branch, so don't worry.

File: docs/en/ingest-management/faq.asciidoc [hunk 1]
```diff
@@ -233,7 +233,7 @@ are both accessible; by default these are `5601` and `9200` respectively.
 [[how-do-i-reinstall-a-missing-dashboard-asset]]
 == If I delete an integration dashboard asset from {kib}, how do I get it back?
 
-To reinstall the assets for a specific integration, you can use the {fleet} API using the package name and version.
+To reinstall the assets for a specific integration, you can use the {fleet} API using the package name and version. This needs to be run against the Kibana API and not the Elasticsearch API to be succesfull.
```
Comment (dedemorton) [inline docs/en/ingest-management/faq.asciidoc:236]:
```suggestion
To reinstall the assets for a specific integration, you can use the {fleet} API using the package name and version. This needs to be run against the {kib} API and not the {es} API to be succesfull. 
```

**Review by dedemorton — APPROVED** [review]
LGTM. Go ahead and merge, and I will make sure this gets backported to relevant branches.

**Pull request merged**